backend/code/m-booking/src/main/java/com/mbooking/model/Customer.java

```diff
@@ -14,8 +14,7 @@
 
 @Entity
 @DiscriminatorValue("CUSTOMER")
-@Getter
```

Comment:
Can't seem to figure out what's the difference now. It would be nice if we stayed consistent, and all used lombok instead of the old fashioned getters and setters.

Reply:
Ovde sam imala neki problem sa geterima oko pokretanja testova pa sam zato probala drugacije